pkg/ebpf/tracee.go

```diff
@@ -362,12 +362,16 @@ func (t *Tracee) Init(ctx gocontext.Context) error {
 
 	err = capabilities.GetInstance().Specific(
 		func() error {
-			t.kernelSymbols, err = environment.NewKernelSymbolTable(
-				environment.WithRequiredSymbols(t.requiredKsyms),
-			)
+			t.kernelSymbols = environment.NewKernelSymbolTable(true, true)
+			// t.requiredKsyms may contain non-data symbols, but it doesn't affect the validity of this call
```

Comment:
Since plenty of the symbols i've attempted to include here belong to the `T` symbols, I think trimming that logic will save both RSS, loading time, and code complexity.
If you do it, please do so in a separate commit.

Reply:
I could trim the symbols included for attachment purposes, but anything included as a ksymbol event dependency needs to stay becuase we don't know if it's a data symbol or not. I think the effect on RSS and loading time would be negligible, but maybe it's worth it for the sake of code complexity.

Reply:
I think that would still leave us with plenty of symbols down. Aren't the mem_dump symbols also mostly `T` symbols?

Reply:
> I think that would still leave us with plenty of symbols down. Aren't the mem_dump symbols also mostly `T` symbols?

We have no way to distinguish between T and D symbols requested by `print_mem_dump` (until they are found in the kallsyms file).